src/bool.cr

```diff
@@ -55,4 +55,9 @@ struct Bool
   def to_s(io)
     io << to_s
   end
+
+  # Returns `self`
```

Comment:
Yeah, I guess no docs is fine too, one should expect `Bool#clone` to return a bool with the same value any way, and saying "returns self" can be misleading indeed.